app/renderer/js/utils/logger-util.js

```diff
@@ -88,6 +95,20 @@ class Logger {
 	reportSentry(err) {
 		captureException(err);
 	}
+
+	trimLog(file) {
+		fs.readFile(file, 'utf8', (err, data) => {
+			if (err) {
+				throw err;
+			}
+			// Keep bottom MAX_LOG_FILE_LINES of each log instance
+			const MAX_LOG_FILE_LINES = 500;
+			const logs = data.split(os.EOL);
```

Comment:
We could probably do a check like `if (logs.length <500)` then don't do the trimming part. Would reduce a bit unnecessary computation and file write.

Reply:
Yeah, good thought ! will do.